### DISCUSSION TITLE
What is the correct way to install in a non-npm application?

### DISCUSSION THREAD

**Discussion by leodip** [saadeghi/daisyui#2405] [Q&A] [answered]
I'm running a Golang application with HTML templates. That means no NPM is available and the HTML pages are rendered server side.

I'm currently using this on my layout page:

```
<script src="https://cdn.jsdelivr.net/npm/daisyui@3.9.2/src/index.min.js"></script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/daisyui@3.9.2/dist/full.min.css">
```

The problem is the javascript file. If I include it, I get this error:

```
Uncaught ReferenceError: require is not defined
    <anonymous> index.js:1
index.js:1:23
```

Will it work as expected if I remove the javascript declaration? Or is there a way to include the javascript without getting the error?

Thank you

**Comment by saadeghi** [comment, marked as answer]
You don't need to include that JS file. It's for build time, not the browser.  
`full.min.css` is enough and it includes all the class names. 

However for an non-Node environment a challenge would be purging unused class names because otherwise the CSS file is going to be large (because it includes all the class names you didn't use as well)

I'm not sure if there's good tool to purge unused styles in Go but I found this one: https://github.com/client9/csstool

**Comment by leodip** [comment]
Understood, thanks a lot for your reply!
